### PR TITLE
VIH-10090 Update JOH in the booking

### PR DESCRIPTION
**Pull request by oliver-scott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-10090

### Change description ###

- Pass in the user role, hearing role and case type group when updating participants so that we can modify a participant's role for the update JOH endpoint

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: BookingQueueSubscriber/BookingQueueSubscriber.Services/BookingQueueSubscriber.Services.csproj [hunk 1]
```diff
@@ -23,7 +23,7 @@
       </PackageReference>
       <PackageReference Include="TimeZoneConverter" Version="3.3.0" />
       <PackageReference Include="UserApi.Client" Version="1.45.2" />
-      <PackageReference Include="VideoApi.Client" Version="1.44.6" />
+      <PackageReference Include="VideoApi.Client" Version="1.44.16-pr-0599-0004" />
```
Comment (oliver-scott) [inline BookingQueueSubscriber/BookingQueueSubscriber.Services/BookingQueueSubscriber.Services.csproj:26]: TODO update after merging video api

File: BookingQueueSubscriber/BookingQueueSubscriber.Services/packages.lock.json [hunk 2]
```diff
@@ -74,9 +74,9 @@
       },
       "VideoApi.Client": {
         "type": "Direct",
-        "requested": "[1.44.6, )",
-        "resolved": "1.44.6",
-        "contentHash": "gMcnXaysO+kT390Ehc9IZBCiek4ec5L9WF9zExkRXh+9EL8RW5duiQIlDprD5iIfpp/YmIkAgXo+OW8j7TGlYg==",
+        "requested": "[1.44.16-pr-0599-0004, )",
```
Comment (oliver-scott) [inline BookingQueueSubscriber/BookingQueueSubscriber.Services/packages.lock.json:77]: TODO update after merging video api

**Review by shaed-parkar — APPROVED** [review]

**Review by phildebrah — APPROVED** [review]
Approved

**Review by phildebrah — APPROVED** [review]
Approved

**Pull request merged**